packages/near-api-js/test/providers.test.js

```diff
@@ -295,6 +295,41 @@ test('json rpc light client proof', async() => {
     await expect(provider.lightClientProof(lightClientRequest)).rejects.toThrow(/.+ block .+ is ahead of head block .+/);
 });
 
+test('json rpc get next light client block', withProvider(async (provider) => {
```

Comment:
New tests for this should go in `@near-js/providers` as `near-api-js/tests` will be removed soon. This isn't documented anywhere but these tests only remain to ensure that the migration to packages did not break any test coverage.